templates/ansible/inventory.tmpl

```diff
@@ -73,11 +73,13 @@ nfs_server
 [${prefix}_${server_name}_pgsql]
 ${server_data.server_ip}
 [${prefix}_${server_name}_pgsql:vars]
-postgres_server_name=${server_name}
-postgres_server_version=${server_data.server_version}
-postgres_server_ssl=${server_data.server_ssl}  # NOTE: Values - [on,off]
-postgres_administrator_login=${server_data.administrator_login} # NOTE: Do not change this value at this time
-postgres_administrator_password=${server_data.administrator_password}
+postgres_server_name="${server_name}"
+postgres_server_version="${server_data.server_version}"
+postgres_server_ssl="${server_data.server_ssl}"  # NOTE: Values - [on,off]
+postgres_administrator_login="${server_data.administrator_login}" # NOTE: Do not change this value at this time
```

Comment:
The # NOTE items have been removed from the template. The updates are currently in staging.

Reply:
The notes have been removed, merged staging into this branch to pick up on those changes.